resources/deb/etc/init.d/openhab2

```diff
@@ -71,15 +71,36 @@ fi
 # Load the VERBOSE setting and other rcS variables
 . /lib/init/vars.sh
 
+# Check to see if openHAB is currently running
+EXISTINGPID=`pgrep java -G openhab | tail -1`
```

Comment:
good point! an important topic. maybe this helps http://unix.stackexchange.com/questions/234709/avoid-pgrep-in-sh